src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -110,6 +110,17 @@ void VehicleAirData::SensorCorrectionsUpdate(bool force)
 	}
 }
 
+void VehicleAirData::AirTemperatureUpdate()
+{
+	differential_pressure_s differential_pressure;
+
+	// update air temperature if data from differential pressure sensor is in valid range
+	if (_differential_pressure_sub.update(&differential_pressure) && PX4_ISFINITE(differential_pressure.temperature)
```

Comment:
I guess we could also make this running at a much lower frequency

Reply:
I wouldn't worry about that, these updates are fairly cheap.